cypress_test/integration_tests/mobile/calc/bottom_toolbar_spec.js

```diff
@@ -126,54 +126,6 @@ describe('Interact with bottom toolbar.', function() {
 			.should('have.attr', 'colspan', '1024');
 	});
 
-	it('Apply left/right alignment', function() {
```

Comment:
Why did you remove this test? If it's by accident, could you please restore it? Thanks. CC @pedropintosilva

Reply:
I will get some help to write tests, so hopefully I can define that at calc, impress/draw the align stuff will be available within a submenu. But first I have to learn how to write tests.
In addition I don't know what's the benefit of this tests.